Gemfile

```diff
@@ -40,6 +40,9 @@ gem 'bourbon'
 
 gem 'neat'
 
+gem 'petfinder'
+
```

Comment:
In general you'll find most Gemfiles don't have a lot of extra space around the gem names. It's a good idea to have some separation between the name and the group, but how do you feel about it being a single line instead of two?